inductiva/structures/scenarios/plate_linear_elastic/mesh_utils.py

```diff
@@ -0,0 +1,257 @@
+"""Utils to create the mesh."""
+
+from typing import List
+from typing import Optional
+
+import gmsh
+
+import geometry_utils
+
+
+class Mesh:
+    """Mesh.
+
+    Attributes:
+        geometry: A geometry_utils.GeometricCase object.
+        glb_ref_fac (float): The refinement factor for global refinement of the
+          mesh. 
+        loc_ref_fac (float): The refinement factor for local refinement of  the
+          mesh.
```

Comment:
Say a few more words about these refinement factors. What do they control in specific?